workspace/assets/Gruntfile.js

```diff
@@ -36,7 +36,13 @@ module.exports = function (grunt) {
 	var visitor = (function () {
 		try {
 			return require('./Gruntcustom.js');
-		} catch (e) {}
+		} catch (e) {
+			if (e.code !== 'MODULE_NOT_FOUND') {
+				grunt.verbose.writeln('Failed to load Gruntcustom.js:', e);
```

Comment:
Can we put a distinctive color to this? otherwise we might lose it in the full output :)

Reply:
@fhamon grunt does it for us

Reply:
@fhamon check out 0cbbb1e